src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
+
+  if (days >= 7) {
+    totalCost -= 50;
```

Comment:
Prefer `if` with `return` over `if else` to simplify the code flow. In this case, since you're returning `totalCost` at the end, consider removing the `else` statement and just using consecutive `if` statements.